contracts/crowdsale/emission/AllowanceCrowdsale.sol

```diff
@@ -0,0 +1,30 @@
+pragma solidity ^0.4.18;
+
+import "../Crowdsale.sol";
+import "../../token/ERC20/ERC20.sol";
+import "../../math/SafeMath.sol";
+
+contract AllowanceCrowdsale is Crowdsale {
```

Comment:
Let's try if we can find a better name for this one

Reply:
Oh, I thought it was very clear, even more considering the alternatives @fiiiu and @ajsantander discussed.